internal/postgres/config.go

```diff
@@ -172,16 +172,39 @@ func reloadCommand(name string) []string {
 	// mtimes.
 	// - https://unix.stackexchange.com/a/407383
 	script := fmt.Sprintf(`
+# Parameters for curl when managing autogrow annotation.
+APISERVER="https://kubernetes.default.svc"
+SERVICEACCOUNT="/var/run/secrets/kubernetes.io/serviceaccount"
+NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+TOKEN=$(cat ${SERVICEACCOUNT}/token)
+CACERT=${SERVICEACCOUNT}/ca.crt
+
 declare -r directory=%q
 exec {fd}<> <(:)
 while read -r -t 5 -u "${fd}" || true; do
+  # Manage replication certificate.
   if [ "${directory}" -nt "/proc/self/fd/${fd}" ] &&
     install -D --mode=0600 -t %q "${directory}"/{%s,%s,%s} &&
     pkill -HUP --exact --parent=1 postgres
   then
     exec {fd}>&- && exec {fd}<> <(:)
     stat --format='Loaded certificates dated %%y' "${directory}"
   fi
+
+  # Manage autogrow annotation.
+  # Return size in Mebibytes.
+  size=$(df --human-readable --block-size=M /pgdata | awk 'FNR == 2 {print $2}')
+  use=$(df --human-readable /pgdata | awk 'FNR == 2 {print $5}')
+  sizeInt="${size//M/}"
+  # Use the sed punctuation class, because the shell will not accept the percent sign in an expansion.
+  useInt=$(echo $use | sed 's/[[:punct:]]//g')
+  triggerExpansion="$((useInt > 75))"
+  if [ $triggerExpansion -eq 1 ]; then
+    newSize="$(((sizeInt / 2)+sizeInt))"
+    newSizeMi="${newSize}Mi"
+    d='[{"op": "add", "path": "/metadata/annotations/suggested-pgdata-pvc-size", "value": "'"$newSizeMi"'"}]'
+    curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -XPATCH "${APISERVER}/api/v1/namespaces/${NAMESPACE}/pods/${HOSTNAME}?fieldManager=kubectl-annotate" -H "Content-Type: application/json-patch+json" --data "$d"
```

Comment:
🤔 Thinking we should force ownership here.  I believe this is done via another query param, e.g. `force=true`.

https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts
> **Overwrite value, become sole manager:** If overwriting the value was intentional (or if the applier is an automated process like a controller) the applier should set the force query parameter to true (for kubectl apply, you use the --force-conflicts command line parameter), and make the request again. This forces the operation to succeed, changes the value of the field, and removes the field from all other managers' entries in managedFields.

Reply:
Makes sense. Thanks, Andrew!

Reply:
Not permitted:

```
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "PatchOptions.meta.k8s.io \"\" is invalid: force: Forbidden: may not be specified for non-apply patch",
  "reason": "Invalid",
  "details": {
    "group": "meta.k8s.io",
    "kind": "PatchOptions",
    "causes": [
      {
        "reason": "FieldValueForbidden",
        "message": "Forbidden: may not be specified for non-apply patch",
        "field": "force"
      }
    ]
  },
  "code": 422

```